pyproject.toml

```diff
@@ -194,13 +207,15 @@ version_files = [
 ]
 
 [tool.mypy]
+enable_error_code = ["ignore-without-code", "redundant-expr", "truthy-bool"]
```

Comment:
Respectively:

- Force us to use codes, e.g. `type: ignore[<code here>]`
- e.g. checking truthiness of a value that is always `True`
- i.e. checking truthiness of an object that doesn't implement `__bool__` and thus could always be `True`